eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig

```diff
@@ -290,93 +293,93 @@
 
 {% block settings_maxfilesize %}
     <li class="ez-fielddefinition-setting maximum-file-size">
-        <div class="ez-fielddefinition-setting-name">Maximum file size:</div>
+        <div class="ez-fielddefinition-setting-name">{{ 'fielddefinition.maximum-file-size.label'|trans|desc("Maximum file size:")}}</div>
         <div class="ez-fielddefinition-setting-value">
         {% if fielddefinition.validatorConfiguration.FileSizeValidator.maxFileSize %}
             {# TODO l10n / unit #}
-            {{ fielddefinition.validatorConfiguration.FileSizeValidator.maxFileSize }} bytes
+            {{ fielddefinition.validatorConfiguration.FileSizeValidator.maxFileSize }} {{ 'fielddefinition.maximum-file-size.unit'|trans|desc("bytes")}}
```

Comment:
The `bytes` part has been changed to `MB` in #1900 (beware, it's a gigantic PR), merged a couple minutes ago. Could you rebase on top of it and adapt ?

Also, I believe thazt the maxFileSize should be made a parameter of the string, could you change it ?